ts_conda_build/conda/meta.yaml

```diff
@@ -1,6 +1,9 @@
 package:
   name: ts-conda-build
-  version: 0.3.11
+  version: 0.3.12
```

Comment:
You probably don't need to change the version for this change do you?

Reply:
Agreed, reverted!